TEST-313 Update Jenkins branch

MarkWareham opened the pull request:
from master

Commit:
QA-118 clean up profile names

Cousjava committed:
Merge pull request #1 from MarkWareham/clean-up-profiles

QA-118 clean up profile names

Commit:
fixed duplicate repo IDs

MarkWareham committed:
Merge pull request #2 from MarkWareham/fixed-duplicate-repo-ids

fixed duplicate repo IDs

MattGill98 commented on pom.xml:
```diff
@@ -104,13 +104,13 @@
 		<module>rest-client</module>
         <module>open-api</module>
-		<module>test-utils</module>   
+		<module>test-utils</module>
```
Bleurgh tabs!

MarkWareham replied:
Exists in master! - this PR an update of jenkins branch

MattGill98 commented on pom.xml:
```diff
@@ -303,14 +303,14 @@
 
 		<profile>
-			<id>payara-ci-managed</id>
+			<id>payara-server-managed</id>
```
Bleurgh more tabs!

MarkWareham replied:
Exists in master! - this PR an update of jenkins branch. A trivial issue such as tabs shouldn't stop update. It should have been addressed in the original PR. Not crucial to functionality

juangon committed:
Add Thorntail support

arjantijms committed:
Merge pull request #2 from juangon/thorntail_support

Add Thorntail support

Commit:
resolved merge conflicts

cubastanley committed:
Merge pull request #4 from cubastanley/master

Merged upstream changes [TEST-283]

cubastanley reviewed (approved):
Agreed with matt on those tabs, also considering the state of the updated test causing potential issues although if that remains ignored then all is well

pdudits committed:
PAYARA-3660: Drop bean defining annotation from Rest interface

Since Rest Client 1.1, the registration should work without additional
annotation

pdudits committed:
Merge pull request #5 from pdudits/payara-3660

PAYARA-3660: Drop bean defining annotation from Rest interface

Commit:
Updated dependencies. Runs on JDK 11 now.

Commit:
Merge https://github.com/javaee-samples/microprofile1.4-samples

Commit:
Merge pull request #6 from AlanRoth/master

TEST-329 Sync Test Repos
